src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -118,13 +118,13 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="SetColorSchemeParentCommandName" xml:space="preserve">
-    <value>Select color scheme</value>
+    <value>Change color scheme</value>

```

Comment:
Honestly, I don't think these are better. There's no indication that "Change" means "i will open a menu for you to select from" nor a suggestion that "Select" does not mean that.

Reply:
Referencing some relevant thoughts from #16846:
> For `Select color scheme... >`, I feel like the `...` are wrong because of the `>`, and I think `Change color scheme >` would be clearer as the action is to change the currently active color scheme, not select it to do something with it like you would `Select all text` to then copy it for example.

Hmm. I see both points. Perhaps `Apply` would be the better term here. "Select"-ing a color scheme doesn't necessarily imply what you're doing with the color scheme, only that you're selecting one.

As for the `...`, yeah, the `>` on the command palette makes it clear that the user is going to enter a nested menu, so I'm keeping the `...` removed.